src/Align.jl

```diff
@@ -0,0 +1,15 @@
+# Bio.Align
+# =========
+#
+# Module for biological sequence alignment.
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+module Align
+
+using Reexport
+
+@reexport using BioAlignments
```

Comment:
Ok, I'll have to do that for the other modules too in that case, as up until now reexport has been used for them all.